lib/iris/experimental/ugrid/__init__.py

```diff
@@ -3673,7 +3689,6 @@ def _build_mesh(cf, mesh_var, file_path):
             ", consistent with the attached connectivities."
         )
         # TODO: reconsider logging level when we have consistent practice.
-        # TODO: logger always requires extras['cls'] : can we fix this?
```

Comment:
`extra` is useful for almost all examples in this module, excepting those in `_build_mesh()`. AFAICT the only way to not require this kwarg, while still offering the benefits when wanted, would be something like this at the top of the module:

```Python
logger._log = functools.partial(logger._log, extra=dict(cls=None))
```

... and that feels pretty hacky, right?